### DISCUSSION TITLE
Availability for WikiJS in version 3.0

### DISCUSSION THREAD

**Discussion by craph** [requarks/wiki#4900] [Help / Questions]
Hello,

When WikiJS will be available in version 3.0 ?

I want to migrate my production dokuwiki to WikiJS :) Should I wait the version 3 or can I migrate to version 2.5 ?

Thank you very much.
Best regards,

**Comment by craph** [comment]
up ? !

**Reply by root9191** [reply]
I am also interested when it will be released.

**Comment by craph** [comment]
Hello,

Is it possible to test the version 3 in Beta ?

Thank you
